.tx/config

```diff
@@ -9,7 +9,7 @@ file_filter = MapboxNavigation/Resources/<lang>.lproj/Localizable.strings
 source_file = MapboxNavigation/Resources/Base.lproj/Localizable.strings
 source_lang = en
 
-[mapbox-navigation-core-ios.localizablestrings]
+[mapbox-navigation-ios.LocalizableCoreStrings]
 file_filter = MapboxCoreNavigation/Resources/<lang>.lproj/Localizable.strings
```

Comment:
`[mapbox-navigation-ios.LocalizableCoreStrings]` is now referenced twice in this file. Is this expected?